xml/System/String.xml

```diff
@@ -436,15 +436,15 @@ Use the <xref:System.String.Compare%2A?displayProperty=nameWithType> method to d
 
- The following example illustrates the danger of performing a culture-sensitive comparison for equality when an ordinal one should be used instead. In this case, the intent of the code is to prohibit file system access from URLs that begin with "FILE://" or "file://" by performing a case-insensitive comparison of the beginning of a URL with the string "FILE://". However, if a culture-sensitive comparison is performed using the Turkish (Turkey) culture on a URL that begins with "file://", the comparison for equality fails, because the Turkish uppercase equivalent of the lowercase "i" is "İ" instead of "I". As a result, file system access is inadvertently permitted. On the other hand, if an ordinal comparison is performed, the comparison for equality succeeds, and file system access is denied.
+ The following example illustrates the danger of performing a culture-sensitive comparison for equality when an ordinal one should be used instead. In this case, the intent of the code is to prohibit file system access from URLs that begin with "FILE://" or "file://" by performing a case-insensitive comparison of the beginning of a URL with the string "FILE://". However, if a culture-sensitive comparison is performed using the Turkish (Turkey) culture on a URL that begins with "file://", the comparison for equality fails, because the Turkish uppercase equivalent of the lowercase "i" is "I" instead of "I". As a result, file system access is inadvertently permitted. On the other hand, if an ordinal comparison is performed, the comparison for equality succeeds, and file system access is denied.
 
  :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.String.Class/cpp/string.equals.cpp" id="Snippet11":::
  :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Class/cs/equality1.cs" id="Snippet11":::
  :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.String.Class/vb/equality1.vb" id="Snippet11":::
 
 <a name="Normalization"></a>
 ## Normalization
- Some Unicode characters have multiple representations. For example, any of the following code points can represent the letter "ắ":
+ Some Unicode characters have multiple representations. For example, any of the following code points can represent the letter "?":
```

Comment:
Seems this file also had some unicode characters changed. Most of the changes were in lines that contained such characters, but I don't see too many lines with actual docs porting.